test/integration/users_edit_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class UsersEditTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  test "unsuccessful edit" do
+    get edit_user_path(@user)
+    assert_template 'users/edit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.